src/site/content/en/handbook/self-assessment-components/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: handbook
+title: Self-assessments
+date: 2020-01-28
+description: |
+  Learn how to use web.dev's self-assessment components.
+---
+
+Self-assessments provide opportunities for users
+to check their understanding of concepts covered in your post.
+
+{% Assessment page, 'self-assessment' %}
```

Comment:
On initial readthrough I thought `'self-assessment'` was a required argument to define the _type_ of assessment. But it's really a unique identifier. Maybe rename to `'my-first-assessment'` or something to signify it's just an identifier?

Reply:
Fixed